src/sentry/incidents/models.py

```diff
@@ -91,9 +91,13 @@ def create(self, organization, **kwargs):
             )
 
 
-class IncidentStatus(Enum):
+class IncidentType(Enum):
     DETECTED = 0
     CREATED = 1
+
+
+class IncidentStatus(Enum):
+    OPEN = 1
```

Comment:
We don't need to worry about any statuses of 0, since no one has created a `DETECTED` incident so far.